wordpress/wp-content/plugins/cds-base/classes/Modules/Cleanup/Profile.php

```diff
@@ -128,8 +128,30 @@ public function removeAdditionalCapabilitiesFunc(): bool
         return false;
     }
 
+    public function yoastOptions(): void
+    {
+        // re-add via hidden fields -- with no values
+        // :( we need these values in order to submit without yoast errors
+
+        $fields = ["honorificPrefix",
+        "honorificSuffix",
+        "birthDate",
+        "gender",
+        "award",
+        "knowsAbout",
+        "knowsLanguage",
+        "jobTitle",
+        "worksFor"];
```

Comment:
I can't believe we are taking away "award" after that cool TBS one we got for COVID Alert

Reply:
It's still on for Super Admins @pcraig3 so you can still input your excellence for that field.